### DISCUSSION TITLE
Light theme

### DISCUSSION THREAD

**Discussion by meyt** [solidtime-io/solidtime#234] [Feature requests]
Forcing dark theme is bad idea. Please respect os/browser theme preferences.

**Comment by MaelImhof** [comment]
Having a light theme would indeed be neat. Syncing with the system is also a big plus.

**Comment by Onatcer** [comment]
We decided to move feature requests into github discussions, to separate feature requests from bugs and allow for better prioritization. If this feature is important to you, you can upvote it to let us know.

**Comment by cameronj86** [comment]
Until a day theme is created, a temporary hack for KDE users is to create a shortcut command for the `invert`'s window option under `Desktop Effects`. It's not the prettiest setup but it'll have to do for now:
![image](https://github.com/user-attachments/assets/0f3aa3da-bab8-45c6-9318-c979e368ae67)

**Comment by korridor** [comment]
🎉 The release [v0.7.0](https://github.com/solidtime-io/solidtime/releases/tag/v0.7.0) of [solidtime](https://app.solidtime.io/) now has a light theme
